ultraplot/axes/plot.py

```diff
@@ -396,7 +396,7 @@
     If the latter, levels are inferred using `~ultraplot.utils.edges`.
     This will override any `levels` input.
 center_levels : bool, default False
-    If set to true, the discrete color bar bins will be centered on the level values 
+    If set to true, the discrete color bar bins will be centered on the level values
     instead of using the level values as the edges of the discrete bins. This option can be used for diverging, discrete color bars with both positive and negative data to ensure data near zero is properly represented.
```

Comment:
```suggestion
    If set to true, the discrete color bar bins will be centered on the level values
    instead of using the level values as the edges of the discrete bins. This option
    can be used for diverging, discrete color bars with both positive and negative
    data to ensure data near zero is properly represented.
```